### PR TITLE
[ADD] sales_team_security: Migration and review of module sales_team_…

### PR DESCRIPTION
**Pull request by victor-ditay**
…security from "https://github.com/OCA/sale-workflow/pull/271" OdooV8 to OdooV9

@elicoidal

cc @elicoidal

### CODE REVIEW

File: sales_team_security/README.rst [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+help us smashing it by providing a detailed and welcomed `feedback
+<https://github.com/OCA/
+sale-workflow/issues/new?body=module:%20
+sales_team_security%0Aversion:%20
+9.0%0A%0A**Steps%20to%20reproduce**%0A-%20...%0A%0A**Current%20behavior**%0A%0A**Expected%20behavior**>`_.
+
+Known issues/Roadmap
+====================
+
+* This module is designed for support only sales part, so someone that has
```
Comment (elicoidal) [inline sales_team_security/README.rst:46]: Dont use list if only one element

File: sales_team_security/__init__.py [hunk 2]
```diff
@@ -0,0 +1,5 @@
+# -*- coding: utf-8 -*-
+# © 2016 Pedro M. Baeza <pedro.baeza@serviciosbaeza.com>
```
Comment (elicoidal) [inline sales_team_security/__init__.py:2]: add Elico copyright

**Comment by elicoidal** [comment]
some details. LGTM

**Comment by pedrobaeza** [comment]
commit history hasn't been overwritten, so :-1: while this is not respected.

**Comment by victor-ditay** [comment]
@pedrobaeza Sorry by that.

I'll fix the commit history. By the way I have a question if you could help me:

Is related to Travis test. that fails on test `sale_sourced_by_line`. 
I know this module is not ported but in the `.travis.yml` specify to test this module and others that isn’t ported now.
Should I fix this issue making another Pull Request to fix `.travis.yml` on the master branch OCA/sale-workflow/9.0?

Thanks.

cc @elicoidal

**Comment by pedrobaeza** [comment]
You should clean .travis.yml and remove the lines with EXCLUDE/INCLUDE and let only one general line for odoo and another for OCB.

**Comment by pedrobaeza** [comment]
Closing as the previous version is not yet finished

**Pull request closed without merging**